packages/rate-limit/README.md

```diff
@@ -0,0 +1,32 @@
+# @tinyhttp/pug
+
+[![npm (scoped)][npm-badge]](https://npmjs.com/package/@tinyhttp/pug) [![npm][dl-badge]](https://npmjs.com/package/@tinyhttp/pug) [![][web-badge]](https://tinyhttp.v1rtl.site/mw/pug)
+
+[Pug](https://github.com/pugjs/pug) wrapped in a helper function for tinyhttp.
+
+## Install
+
+```sh
+pnpm i @tinyhttp/pug
+```
+
+## Example
+
+```js
+import { App } from '@tinyhttp/app'
+import { pug } from '@tinyhttp/pug'
+
+const app = new App()
+
+pug(/* Pug options */)(app)
+
+app.get((req, res) => void res.render('index.pug')).listen(3000)
+```
+
+## License
+
+MIT © [v1rtl](https://v1rtl.site)
```

Comment:
also, for the future, you can assign the license to yourself because you wrote the module :D